test/server.jl

```diff
@@ -0,0 +1,24 @@
+# find a good port
+conn = listenany(2228)
+close(conn[2])
+port = conn[1]
+
+@async lintserver(port)
+sleep(1) #let sever start
```

Comment:
Oh yes I see that the internal `@async` is in an infinite loop. Shouldn't you be shutting the server down at the end of the tests?